apps/web/public/static/locales/en/common.json

```diff
@@ -264,6 +264,7 @@
   "add_another_calendar": "Add another calendar",
   "other": "Other",
   "email_sign_in_subject": "Your sign-in link for {{appName}}",
+  "email_all_details": "We emailed you with all the details",
   "emailed_you_and_attendees": "We emailed you and the other attendees a calendar invitation with all the details.",
   "emailed_you_and_attendees_recurring": "We emailed you and the other attendees a calendar invitation for the first of these recurring events.",
   "emailed_you_and_any_other_attendees": "You and any other attendees have been emailed with this information.",
```

Comment:
It also has attendees in it. So, wherever this key is used we need to fix there as well.

Reply:
Did not get this. Can you please explain this?

Reply:
I mean the key 'emailed_you_and_any_other_attendees' also needs to be changed in a similar way.